cra/sample/pr-pipeline.yaml

```diff
@@ -0,0 +1,326 @@
+          value: $(params.pipeline-debug)
+
+    - name: cra-cis-check
+      taskRef:
+        name: cra-cis-check
+      runAfter:
+        - cra-discovery-scan
+      workspaces:
+        - name: secrets
+          workspace: artifacts
+        - name: artifacts
+          workspace: artifacts
+      params:
+        - name: repository
+          value: $(params.repository)
+        - name: revision
+          value: $(params.branch)
+        - name: pr-url
+          value: $(params.pr-url)
+        - name: commit-id
+          value: $(params.commit-id)
+        - name: directory-name
+          value: $(tasks.code-fetch-code.results.directory-name)
+        - name: ibmcloud-api-key
+          value: $(params.api-key)
+        - name: scm-type
+          value: $(params.scm-type)
+        - name: project-id
+          value: $(params.project-id)    
+        - name: pipeline-debug
+          value: $(params.pipeline-debug)
+
+    - name: code-cis-check-status-finished
+      runAfter:
+        - cra-cis-check
+      workspaces:
+        - name: artifacts
+          workspace: artifacts  
+      taskRef:
+        name: git-set-commit-status
+      params:
+        - name: repository
+          value: $(params.repository)
+        - name: revision
+          value: $(params.commit-id)
+        - name: state
+          value: "$(tasks.cra-cis-check.results.status)"
+        - name: context
+          value: cra-cis-scan
+        - name: description
+          value: "Tekton PR Pipeline Status"
+        - name: pipeline-debug
+          value: $(params.pipeline-debug)
+
+    - name: code-bom-check-status-pending
+      runAfter:
+        - code-fetch-code
+      workspaces:
+        - name: artifacts
+          workspace: artifacts  
+      taskRef:
+        name: git-set-commit-status
+      params:
+        - name: repository
+          value: $(params.repository)
+        - name: revision
+          value: $(params.commit-id)
+        - name: state
+          value: pending
+        - name: context
+          value: cra-bom-scan
+        - name: description
+          value: "Tekton PR Pipeline Status"
+        - name: pipeline-debug
+          value: $(params.pipeline-debug)
+          
+    - name: cra-bom
+      taskRef:
+        name: cra-bom
+      runAfter:
+        - cra-discovery-scan
+      workspaces:
+        - name: artifacts
+          workspace: artifacts
+        - name: secrets
+          workspace: artifacts          
+      params:
+        - name: repository
+          value: $(params.repository)
+        - name: revision
+          value: $(params.branch)
+        - name: pr-url
+          value: $(params.pr-url)
+        - name: commit-id
+          value: $(params.commit-id)
+        - name: target-branch
+          value: $(params.target-branch)
+        - name: target-commit-id
+          value: $(params.target-commit-id)      
+        - name: ibmcloud-api-key
+          value: $(params.api-key)
```

Comment:
Hum... are we dealing here with the real value for api key to use ? not the indirection in the secret secure-properties by default ?